package-lock.json

```diff
@@ -1,10030 +1,8 @@
 {
   "name": "prisoner-content-hub-frontend",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
We've not added to the package. To avoid package-lock tennis, we need to sync our node and npm versions I think 🤔